msal/tests/Test.MSAL.NET.Common/Core/Helpers/CoreTestConstants.cs

```diff
@@ -46,6 +46,8 @@ class CoreTestConstants
         public static readonly string PrefCacheAuthorityCommonTenant = "https://" + ProductionPrefCacheEnvironment + "/common/";
         public static readonly string AuthorityOrganizationsTenant = "https://" + ProductionPrefNetworkEnvironment + "/organizations/";
         public const string B2CAuthority = "https://login.microsoftonline.in/tfp/tenant/policy/";
+        public const string B2CLoginAuthority = "https://sometenantid.b2clogin.com/tfp/sometenantid/policy/";
+        public const string B2CInvalidAuthority = "https://sometenantid.b2blogin.com/tfp/sometenantid/policy/";
```

Comment:
>B2CInvalidAuthority [](start = 28, length = 19)

these 2 authorities looks very similar, not sure why one is Invalid

Reply:
i fixed the name... this one is not in the list of trusted hosts, so if you were to pass ValidateAuthority=true, it would fail. We can discuss this further if you want. PS. this one has b2b instead of b2c.

---
In reply to: [230836424](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1383#discussion_r230836424) [](ancestors = 230836424)